src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/Microsoft.DotNet.UpgradeAssistant.Steps.Packages.csproj

```diff
@@ -1,6 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- TODO: Enable once analysis is done -->
+    <NoWarn>$(NoWarn);DV0001</NoWarn>
```

Comment:
What is this for?

Reply:
@sunandabalu introduced a dependency here that is broken; she's fixed it in a separate PR that is currently open. This disables that check